Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs

```diff
@@ -922,23 +927,23 @@ private class ParameterSetNames
 
         /// <summary>
         /// <para type="description">
-        /// The DatabaseInstance that describes the Replica we want to promote.
+        /// The id of the BackupRun to restore to. 
         /// </para>
         /// </summary>
         [Parameter(Mandatory = true, Position = 0)]
         public long BackupRunId { get; set; }
 
         /// <summary>
         /// <para type="description">
-        /// The name/ID of the Replica resource to promote.
+        /// The name/ID of Instance we are backing up. 
```

Comment:
We are backing up this instance? I thought we were changing it's data to that of the backup?